pom.xml

```diff
@@ -86,6 +86,18 @@
       <version>1.0.0</version>
       <scope>test</scope>
     </dependency>
+        <dependency>
+    	<groupId>commons-io</groupId>
```

Comment:
Quick note just for future reference: I realize this is just for `test` scope, but I try to be careful for limiting number of dependencies added, so I'll change code not to need this.